src/applications/personalization/view-representative/components/SideBar.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Telephone, {
+  CONTACTS,
+} from '@department-of-veterans-affairs/formation-react/Telephone';
+
+const SideBar = () => {
```

Comment:
I wonder if we should use the sidebar component from rated disabilities on this? I know there was discussion around getting rid of these sidebars completely

Reply:
I think it's fine here the way it is. There are some slight differences, but this one is largely based off the one we have in View Dependents.